src/Spinners/LoadSpinner.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {
+  FunctionComponent, ReactElement,
+} from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleNotch } from '@fortawesome/free-solid-svg-icons';
+import LoadSpinnerText from './LoadSpinnerText';
+
+export interface LoadSpinnerProps {
+  /** Specifies the text under the spinner */
+  children: string;
+  /** Specifies the color of text under the spinner */
+  color?: string;
+}
+
+const StyledLoadSpinner = styled.div`
+  background: transparent;
+  color: lightblue;
```

Comment:
This should use one of our defined color constants, probably `theme.color.background.info.medium`